content/en/tracing/trace_collection/compatibility/cpp.md

```diff
@@ -18,15 +18,13 @@ further_reading:
 
 The C++ Datadog Trace library is open source - view the [GitHub repository][1] for more information.
 
-This library requires C++14 to build, but if you use [dynamic loading][2], then OpenTracing requires [C++11 or later][3].
+This library requires C++17 to build.
 
-Supported platforms include Linux and Mac. To request Windows support, [contact Datadog support][4].
+Supported platforms include Linux and Mac. To request Windows support, [contact Datadog support][2].
```

Comment:
Can we also add supported architectures here, as implied by the [other doc](https://github.com/DataDog/documentation/pull/21925/files#diff-308a47aecdbf8bd2c834a92a21d7b240650d1ec200998367426dbaa34b979da2R31)?

Reply:
Done.